tests/test_training_utils.py

```diff
@@ -0,0 +1,109 @@
+"""Functions used in test_training."""
```

Comment:
Let's rename this file as `training_utils.py` since this file does not contain any tests.